test/test.proto

```diff
@@ -19,6 +22,14 @@ message Order {
         uint32 in = 4;
         Foo foo = 5;
     }
+
```

Comment:
You might have tested this already, but could we add the case of an optional field with stars? I would expect the generated code to print `key: ***` if present, and skip the key if absent. Is that what is currently happening?